steel/src/config.rs

```diff
@@ -86,6 +101,55 @@ impl ChainSpec {
     }
 }
 
+/// A simple structured hasher.
```

Comment:
```suggestion
// NOTE: We do not want to make this public, to avoid having multiple traits with the `digest` function in
// the RISC Zero ecosystem of crates.
/// A simple structured hasher.
```